### ISSUE TITLE
input in md-radio-button is not usable

### ISSUE THREAD

**Issue by norkunas** [angular/material#6146]
http://codepen.io/norkunas/pen/mVypVv

**Comment by DevVersion** [comment]
That's happening through we disable clicking on everything that's beneath the `radio-button`.
@ThomasBurleson Should we allow clicking in there / clicking for choosing the `radio-button` will still work?

**Comment by norkunas** [comment]
Ok, but why?

**Comment by DevVersion** [comment]
We disabling them to allow selecting the radio through clicking on the (for example) text

**Comment by marcysutton** [comment]
Personally, I think it makes more sense to use CSS positioning and keep inputs outside of `<md-radio-group>`. Although it seems like a fine container element, putting an input inside of a radio group is some serious mixing of semantics. The wrapping element is party there for ARIA/accessibility purposes (requiring `radiogroup` on a common parent). Is there a way you can accomplish your design with separate components and CSS instead?

**Comment by norkunas** [comment]
Ok, i'll do it with custom css then. Thank you for clarification.

**Comment by kunKun-tx** [comment]
@marcysutton, I am trying to achieve a similar result. Separate components and CSS will work, but once you enter any text in the separated input box, somehow that raido button losts its md-check. My current workaround involves creating an additional variable for both of these, outside of the original radio-box array.

**Comment by BinaryWasteland** [comment]
Hey @norkunas did you end up getting this working with CSS? If so are you able to share a plunkr or codepen of your end result? I am interested to see how it turned out and if it is responsive

**Comment by norkunas** [comment]
@BinaryWasteland I just don't use this currently.

**Comment by tbone849** [comment]
I know this is a closed and older thread, but I was able to get inputs to work by adding a div with the md-radio-button directly after the md-radio-button and floating the radio button. I made modifications to the original codepen to show it does indeed work. You do have to mess around with margin spacing on the inputs to get them to match up beside the radio button. Pretty hacky and I'm not sure how nicely this plays with accessibility.

[Example](https://codepen.io/tbone849/pen/RmdmZM)
